packages/react-redux/src/hooks/use-all-feature-toggles/use-all-feature-toggles.ts

```diff
@@ -1,30 +1,20 @@
-import { getIsFeatureEnabled, useAdapterContext } from '@flopflip/react';
-import { type TFlags } from '@flopflip/types';
+import { type TAdapterIdentifiers, type TFlags } from '@flopflip/types';
 import { useSelector } from 'react-redux';
 
 import { selectFlags } from '../../ducks/flags';
 
-export default function useAllFeatureToggles(): TFlags {
+export default function useAllFeatureToggles(
+  adapterIdentifiers: TAdapterIdentifiers
```

Comment:
I'm not sure about this API, maybe because I don't have enough knowledge about the library.
The way I understood it, we can have several adapters (in-memory, launchdarkly,...) so I was expecting, as a consumer, to not care about the adapters but just get everything that has been configured (locally or remotely).
Does all the flags get consolidated in the `memory` adapter?

Reply:
Yes, that's the crux of it all. Yes, in a way consumers don't want to care about the actual adapter they get a flag by. You might have multiple and given that the order your combine them determines the order of precedence for flags pulled out of them (in case of duplication).

In this case I am not so sure if "not knowing" makes sense. As the hook is sort of like an export-style hook I felt you might want to be able to pull out one (or actually all) adapter's data and actually be able to care. In the end you don't have to if you pass all adapter identifiers but at least you have to make that decision here.